plugin/trino-opa/src/test/java/io/trino/plugin/opa/ResponseTest.java

```diff
@@ -0,0 +1,133 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.opa;
+
+import com.google.common.collect.ImmutableList;
+import io.airlift.json.JsonCodec;
+import io.airlift.json.JsonCodecFactory;
+import io.trino.plugin.opa.schema.OpaBatchQueryResult;
+import io.trino.plugin.opa.schema.OpaQueryResult;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class ResponseTest
+{
+    private JsonCodec<OpaQueryResult> responseCodec;
+    private JsonCodec<OpaBatchQueryResult> batchResponseCodec;
+
+    @BeforeEach
+    public void setupParser()
+    {
+        this.responseCodec = new JsonCodecFactory().jsonCodec(OpaQueryResult.class);
+        this.batchResponseCodec = new JsonCodecFactory().jsonCodec(OpaBatchQueryResult.class);
+    }
+
+    @ParameterizedTest
+    @ValueSource(booleans = {false, true})
+    public void testCanDeserializeOpaSingleResponse(boolean response)
```

Comment:
Avoid `@ParameterizedTest`, but definitely don't use it with a boolean.  Just do this.

```suggestion
    @Test
    public void testCanDeserializeOpaSingleResponse()
    {
        testCanDeserializeOpaSingleResponse(true);
        testCanDeserializeOpaSingleResponse(false);
    }

    private void testCanDeserializeOpaSingleResponse(boolean response)
```